themes/default/views/admin/orders/show.blade.php

```diff
@@ -2,249 +2,282 @@
-                    <a href="{{ route('admin.clients.edit',$order->user->id) }}"
-                        class="text-blue-500 hover:text-blue-700 dark:text-blue-300 dark:hover:text-blue-500">
-                        {{ $order->user->name }}
-                    </a>
-                </div>
-            </div>
-            <div class="flex flex-row justify-between">
-                <div class="flex flex-col">
-                    <span class="font-bold">{{ __('Created') }}:</span>
-                    <span>{{ $order->created_at }}</span>
-                </div>
-                <div class="flex flex-col">
-                    <span class="font-bold">{{ __('Updated') }}:</span>
-                    <span>{{ $order->updated_at }}</span>
-                </div>
-            </div>
-            <div class="flex flex-row justify-between">
-                @isset($order->coupon()->get()->first()->code)
-                    <div class="flex flex-col">
-                        <span class="font-bold">{{ __('Coupon') }}:</span>
-                        <span>
-                            <a href="{{ route('admin.coupons.edit',$order->coupon()->get()->first()->id) }}"
-                                class="text-blue-500 hover:text-blue-700 dark:text-blue-300 dark:hover:text-blue-500">
-                                {{ $order->coupon()->get()->first()->code }}</a></span>
-                    </div>
-                @endisset
+
+    <div class="grid grid-cols-1 gap-4 w-full">
+        <div class="text-gray-500 dark:text-darkmodetext dark:bg-secondary-100">
+            <div class="flex flex-col lg:flex-row gap-x-4 items-baseline">
+                <x-input disabled type="text" name="client" :label="__('Client')" name="title" value="{{ $order->user->name }}" class="w-full mt-2 lg:mt-0" icon="ri-user-line" />
+
+                <x-input disabled type="text" name="total" :label="__('Total')" icon="ri-money-dollar-circle-line" class="w-full mt-2 lg:mt-0" value="{{ ucfirst($order->total()) }} {{ config('settings::currency_sign') }}"/>
+
+                <x-input disabled type="text" name="created_at" :label="__('Created At')" icon="ri-calendar-line" class="w-full mt-2 lg:mt-0" value="{{ $order->created_at }}"/>
+
+                <x-input disabled type="text" name="updated_at" :label="__('Updated At')" icon="ri-calendar-line" class="w-full mt-2 lg:mt-0" value="{{ $order->updated_at }}"/>
             </div>
         </div>
-        <div class="flex flex-col mt-4">
-            <span class="font-bold">{{ __('Products') }}:</span>
-            <div class="flex flex-col">
+    </div>
+
+    <div class="text-gray-500 dark:text-darkmodetext mt-12 w-full">
+        <div class="flex flex-col mt-4 overflow-x-auto">
+            <h3 class="text-xl leading-5 font-bold dark:text-darkmodetext">
+                {{ __('Products') }}
+            </h3>
+            <style>
+                th, td {
+                    padding: 5px;
```

Comment:
Using tailwindcss classes. If you need specific values you can use `p-[5px]`

Reply:
I know, this part of the code is not mine. I will correct it when I get home.